lib/tic_tac_toe/web/router.ex

```diff
@@ -1,62 +1,54 @@
 defmodule TicTacToe.Web.Router do
   use Plug.Router
-  import TicTacToe.Web.GameSessionPlug
-  alias TicTacToe.Web.GameSessionPlug
+  alias TicTacToe.Web.GameStateStringifier
   alias TicTacToe.Web.View
 
-  @secret String.duplicate("abcdef0123456789", 8)
-
-  plug Plug.Session, store: :cookie, key: "_my_app_session",
-                     encryption_salt: "cookie store encryption salt",
-                     signing_salt: "cookie store signing salt",
-                     key_length: 64, log: :debug
-
   plug Plug.Parsers, parsers: [:urlencoded, :multipart]
   plug Plug.Static, at: "/public", from: :elixir_web_ttt
-  plug :put_secret_key_base
   plug Plug.Logger
   plug :match
-  plug :create_or_find_game
   plug :dispatch
 
-  get ("/tictactoe/options") do
-    response_body = View.game_options()
+  get ("/ttt/options") do
+    response_body = View.stringified_game_options()
     conn |> put_resp_content_type("html") |> resp(200, response_body)
   end
 
-  get ("/tictactoe/play") do
-    game_state = conn |> GameSessionPlug.get_game_state()
-    response_body = View.render_game(game_state)
+  get ("ttt/play/:stringy_game_state") do
+    game_state = GameStateStringifier.parse(stringy_game_state)
+    response_body = View.render_game(game_state, stringy_game_state)
     conn |> put_resp_content_type("html") |> resp(200, response_body)
   end
 
-  get ("/tictactoe/computer_move") do
-    conn |> GameSessionPlug.make_next_move() |> redirect_to("/tictactoe/play")
+  get ("/ttt/computer_move/:stringy_game_state") do
```

Comment:
The query parameter looks a little verbose;

`http://localhost:4000/ttt/play/%7B%22__struct__%22:%22Elixir.Game%22,%22board%22:%7B%22__struct__%22:%22Elixir.Board%22,%22cells%22:[%22x%22,%22empty%22,%22empty%22,%22empty%22,%22o%22,%22empty%22,%22empty%22,%22empty%22,%22empty%22]%7D,%22players%22:[%7B%22__struct__%22:%22Elixir.Player.Human%22,%22io%22:[%22Elixir.UI.Console%22,%22stdio%22],%22mark%22:%22x%22%7D,%7B%22__struct__%22:%22Elixir.Player.MiniMax%22,%22mark%22:%22o%22%7D]%7D`

The query parameters are user facing, so I'd try and make them more "user friendly" if you can call it that, i.e. /play-game?game-type=hvc&board=EXOXXOOXX